pyface/undo/abstract_command.py

```diff
@@ -0,0 +1,76 @@
+# ------------------------------------------------------------------------------
```

Comment:
We need to add an additional enthought copyright header here which starts from the year 2007. This applies for this file and other files which have the Riverbank computing limited copyright header.